src/apps/competitions/tasks.py

```diff
@@ -239,6 +251,9 @@ def get_data_key(obj, file_type, temp_directory, creator):
 def _get_datetime(field):
     if not field:
         return None
+    elif isinstance(field, datetime.date):
+        # turn the date into a datetime @ midnight that day
+        field = datetime.datetime.combine(datetime.date.today(), datetime.time())
```

Comment:
should this be `field = datetime.datetime.combine(field, datetime.time())` Otherwise I think this will always set field to be whatever day it was added, irrespective of the field value?

Reply:
Lame, added tests so this shouldn't happen again -- don't want to spend a ton of time making them full proof but lmk if there are a couple other quick cases we should maybe add